qt/MainWindow.cc

```diff
@@ -1244,6 +1245,12 @@ void MainWindow::refreshPref(int key)
             break;
         }
 
+    case Prefs::READ_CLIPBOARD:
+        {
+            b = prefs_.getBool(Prefs::READ_CLIPBOARD);
+            read_from_clipboard_ = b;
+        }
```

Comment:
There should be a `break;` instead of falling through to the default break.

The effect is the same either way, but adding an explicit `break;` makes it clear that READ_CLIPBOARD is not meant to fall through to anything in case the code in `default:` gets changed in the future